librz/analysis/var_global.c

```diff
@@ -62,17 +75,42 @@ RZ_API RZ_OWN bool rz_analysis_var_global_add(RzAnalysis *analysis, RZ_NONNULL R
  * \param glob Global variable instance
  * \return void
  */
-RZ_API void rz_analysis_var_global_free(RzAnalysisVarGlobal *glob) {
+RZ_API void rz_analysis_var_global_free(RZ_NONNULL RzAnalysisVarGlobal *glob) {
```

Comment:
```suggestion
RZ_API void rz_analysis_var_global_free(RzAnalysisVarGlobal *glob) {
```
NULL is acceptable in a free function.